projects/fmcdaq2/src/devices/adi_hal/parameters.h

```diff
@@ -48,6 +48,7 @@
 #endif
 
 #ifndef ALTERA_PLATFORM
+#define UART_BAUDRATE                           115200
```

Comment:
nit: I think this can be moved outside the platform specific area.

Reply:
there is not iio support for altera, but will move anyway